qa/L0_parameters/test.sh

```diff
@@ -73,7 +73,28 @@ kill $SERVER_PID
 wait $SERVER_PID
 
 export TEST_HEADER=1
-SERVER_ARGS="--model-repository=model_repository --exit-timeout-secs=120 --grpc-header-forward-pattern my_header.* --http-header-forward-pattern my_header.*"
+SERVER_ARGS="--model-repository=model_repository --exit-timeout-secs=120 --grpc-header-forward-prefix my_header --http-header-forward-prefix my_header"
+run_server
+if [ "$SERVER_PID" == "0" ]; then
+    echo -e "\n***\n*** Failed to start $SERVER\n***"
+    cat $SERVER_LOG
+    exit 1
+fi
+
+set +e
+python3 $TEST_SCRIPT_PY >$CLIENT_LOG 2>&1
+if [ $? -ne 0 ]; then
+    cat $CLIENT_LOG
+    echo -e "\n***\n*** Test Failed\n***"
+    RET=1
+fi
+set -e
+
+kill $SERVER_PID
+wait $SERVER_PID
+
+export TEST_HEADER=all
```

Comment:
If only used by the unit test script, perhaps put it as scoped environment variable:
```
TEST_HEADER=all python3 $TEST_SCRIPT_PY >$CLIENT_LOG 2>&1
```
You may also consider to represent them to be different test cases:
```
class CommonTestUtils(tu.TestResultCollector):
    ...

class ForwardPrefixTest(CommonTestUtils):
    ...

class ForwardAllTest(CommonTestUtils):
    ...
```
And invoke the desired test case directly:
```
python3 $TEST_SCRIPT_PY ForwardAllTest >$CLIENT_LOG 2>&1
```

Reply:
This test is removed.